CHANGELOG.md

```diff
@@ -3,6 +3,93 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.16.1
```

Comment:
IMHO, given at least Tree View and Pickers changes this should be a minor release. 🤔 
```suggestion
## 7.17.0
```